charts/pega/values-large.yaml

```diff
@@ -407,3 +409,38 @@ hazelcast:
   # username,password parameters in External Secrets Manager, and enter the external secret for the credentials
   # make sure the keys in the secret should be HZ_CS_AUTH_USERNAME , HZ_CS_AUTH_PASSWORD
   external_secret_name: ""
+
+# Stream (External Kafka) settings.
+stream:
+  # Disabled by default, you must not use stream tier when this setting is enabled.
+  enabled: false
+  # Provide external kafka broker urls.
+  bootstrapServer: ""
+  # Provide Security Protocol used to communicate with brokers. Supported values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL.
+  securityProtocol: PLAINTEXT
+  # If required, provide a Kubernetes secret name that holds the trustStore certificate
+  # Please add the Kubernetes secret name in 'global.certificatesSecrets' too
+  # Pega supports only Java Key Store (.jks) format for truststore.
+  trustStore: ""
```

Comment:
@girirajguptagithub  Is it required to provide kubernetes secret at both the places? here and in global.certificatesSecrets?

Reply:
we need to know the name of the certificate file so that we can set the path appropriately

Reply:
I think the comment is misleading, certificate file name goes here and the secret name goes in global.certificatessecrets.... making a change